py/DREAM/Settings/Equations/RunawayElectrons.py

```diff
@@ -91,6 +98,16 @@ def __init__(self, settings, density=0, radius=0, avalanche=AVALANCHE_MODE_NEGLE
         self.density = None
         self.radius  = None
         self.setInitialProfile(density=density, radius=radius)
+        
+        self.hottail_T_final = np.array([0])
+        self.hottail_T_final_r  = np.array([0])
```

Comment:
These two lines are remnants from the hot-tail low-Z branch and should be deleted.